config/set/code-quality.php

```diff
@@ -2,191 +2,15 @@
-use Rector\CodeQuality\Rector\If_\SimplifyIfNotNullReturnRector;
-use Rector\CodeQuality\Rector\If_\SimplifyIfNullableReturnRector;
-use Rector\CodeQuality\Rector\If_\SimplifyIfReturnBoolRector;
-use Rector\CodeQuality\Rector\Include_\AbsolutizeRequireAndIncludePathRector;
-use Rector\CodeQuality\Rector\Isset_\IssetOnPropertyObjectToPropertyExistsRector;
-use Rector\CodeQuality\Rector\LogicalAnd\AndAssignsToSeparateLinesRector;
-use Rector\CodeQuality\Rector\LogicalAnd\LogicalToBooleanRector;
-use Rector\CodeQuality\Rector\New_\NewStaticToNewSelfRector;
-use Rector\CodeQuality\Rector\NotEqual\CommonNotEqualRector;
-use Rector\CodeQuality\Rector\NullsafeMethodCall\CleanupUnneededNullsafeOperatorRector;
-use Rector\CodeQuality\Rector\Switch_\SingularSwitchToIfRector;
-use Rector\CodeQuality\Rector\Switch_\SwitchTrueToIfRector;
-use Rector\CodeQuality\Rector\Ternary\ArrayKeyExistsTernaryThenValueToCoalescingRector;
-use Rector\CodeQuality\Rector\Ternary\NumberCompareToMaxFuncCallRector;
-use Rector\CodeQuality\Rector\Ternary\SimplifyTautologyTernaryRector;
-use Rector\CodeQuality\Rector\Ternary\SwitchNegatedTernaryRector;
-use Rector\CodeQuality\Rector\Ternary\TernaryEmptyArrayArrayDimFetchToCoalesceRector;
-use Rector\CodeQuality\Rector\Ternary\UnnecessaryTernaryExpressionRector;
-use Rector\CodingStyle\Rector\ClassMethod\FuncGetArgsToVariadicParamRector;
-use Rector\CodingStyle\Rector\FuncCall\CallUserFuncToMethodCallRector;
-use Rector\CodingStyle\Rector\FuncCall\CountArrayToEmptyArrayComparisonRector;
+use Rector\Config\Level\CodeQualityLevel;
 use Rector\Config\RectorConfig;
-use Rector\Php52\Rector\Property\VarToPublicPropertyRector;
-use Rector\Php71\Rector\FuncCall\RemoveExtraParametersRector;
-use Rector\Renaming\Rector\FuncCall\RenameFunctionRector;
-use Rector\Strict\Rector\Empty_\DisallowedEmptyRuleFixerRector;
 
 return static function (RectorConfig $rectorConfig): void {
-    $rectorConfig
-        ->ruleWithConfiguration(RenameFunctionRector::class, [
-            'split' => 'explode',
-            'join' => 'implode',
-            'sizeof' => 'count',
-            # https://www.php.net/manual/en/aliases.php
-            'chop' => 'rtrim',
-            'doubleval' => 'floatval',
-            'gzputs' => 'gzwrites',
-            'fputs' => 'fwrite',
-            'ini_alter' => 'ini_set',
-            'is_double' => 'is_float',
-            'is_integer' => 'is_int',
-            'is_long' => 'is_int',
-            'is_real' => 'is_float',
-            'is_writeable' => 'is_writable',
-            'key_exists' => 'array_key_exists',
-            'pos' => 'current',
-            'strchr' => 'strstr',
-            # mb
-            'mbstrcut' => 'mb_strcut',
-            'mbstrlen' => 'mb_strlen',
-            'mbstrpos' => 'mb_strpos',
-            'mbstrrpos' => 'mb_strrpos',
-            'mbsubstr' => 'mb_substr',
-        ]);
+    foreach (CodeQualityLevel::RULES_WITH_CONFIGURATION as $rectorClass => $configuration) {
```

Comment:
Currently there is only one rule with configuration but I made this future proof by allowing to have an array of them in the `CodeQualityLevel` class